test/test_autograd.py

```diff
@@ -1172,7 +1175,7 @@ def test_attribute_deletion(self):
         with self.assertRaises(RuntimeError):
             del x.data
         with self.assertRaises(TypeError):
-            x.data = None
+            x = None
```

Comment:
That does not raise any error right? This should fail.
And you want to keep the original `.data` here as we explicitely want to check that its behavior did not change.